samples/snippets/pom.xml

```diff
@@ -23,26 +23,12 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
-
-  <!-- [START analytics-admin_install_with_bom] -->
```

Comment:
we can't use the bom until this library is in google-cloud-bom and arguably that shouldn't happen until this is 1.0